multilog/README.md

```diff
@@ -25,18 +25,15 @@ have gotten corrupted on that memory.
 
 ### Setup instructions
 
-Install Verus from `https://github.com/verus-lang/verus`.
+1. Install Verus from `https://github.com/verus-lang/verus`.
+2. **Optional**: If you are on Linux or WSL2 and would like to run CapybaraKV on (real or emulated) PM via Linux's DAX feature, run the following to install PMDK to access PM and LLVM/Clang to generate Rust bindings to PMDK: `sudo apt install libpmem1 libpmemlog1 libpmem-dev libpmemlog-dev llvm-dev clang libclang-dev`
 
-### Verification
 
-To verify the multilog, run the following.
+### Verification
 
-```
-cd deps_hack
-cargo build
-cd ../multilog/src
-verus lib.rs --compile --expand-errors -L dependency=../../deps_hack/target/debug/deps --extern=deps_hack=../../deps_hack/target/debug/libdeps_hack.rlib
-```
+To verify the multilog, make sure Verus (usually at `verus/source/target-verus/release`) is in your path and run one of the following, depending on your setup:
```

Comment:
I suggest saying here that these commands are meant to be run from the `multilog` subdirectory of the `multilog` directory that this README is in.